tags/2.0/_index.md

```diff
@@ -10,17 +10,18 @@ existing custom tags with Jakarta Standard Tag Library tags.
 
 * [Jakarta Standard Tag Library 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jstl/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Standard Tag Library 2.0 Specification Document](./jstl-spec-2.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Standard Tag Library 2.0 Specification Document](./jstl-spec-2.0-SNAPSHOT.html) (HTML)
+* [Jakarta Standard Tag Library 2.0 Specification Document](./jakarta-tags-spec-2.0.pdf) (PDF)
+* [Jakarta Standard Tag Library 2.0 Specification Document](./jakarta-tags-spec-2.0.html) (HTML)
 * [Jakarta Standard Tag Library 2.0 Javadoc](./apidocs)
-* [Jakarta Standard Tag Library 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Standard Tag Library 2.0 Tagdoc](./tagdocs)
+* [Jakarta Standard Tag Library 2.0 TCK](https://download.eclipse.org/jakartaee/tags/jakarta-tags-tck-2.0.0.zip)([sig](https://download.eclipse.org/jakartaee/tags/jakarta-tags-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/tags/jakarta-tags-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/2.0.0-RC1/jar)
+  * [jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/2.0.0/jar)
 
 
 # Compatible Implementations
 
-* [Eclipse Glassfish 6.0.0-M1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads)
+* [Eclipse Glassfish 6.0.0-SNAPSHOT-2020-09-20](https://download.eclipse.org/ee4j/glassfish/weekly/glassfish-6.0.0-SNAPSHOT-2020-09-20.zip)
```

Comment:
This GF link no longer seems to work. And, it's not consistent with the Certification Request that points at a Tags 2.0 implementation...

Reply:
Thanks, it's updated now!